internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -495,7 +495,7 @@ if ($config_path -Eq "") {
     if (Test-Path -Path "$old_config_path") {
         $config_path = $old_config_path
     } else {
-        throw "The installed splunk-otel-collector package does not include a supported config file!"
+        throw "Valid Collector configuration file not found. Rerun the script with Administrator rights."
```

Comment:
@pjanotti Would this be accurate as a message/solution?

Reply:
Correct, in the case, at hand, the MSI fails to install but the error only bubbles up when checking for the config. I was guessing that MSI failure was due to the lack of rights, but, given the `check_if_admin` that doesn't seem to be the case.

Reply:
Can we leave the suggested change as is then?

Reply:
> While this was my initial suspicion for the case at hand, there is a check above if the script is running as admin or not, unless the check is incorrect it seems that this case was already accounted for. It will be better to validate the result of running the MSI, i.e.: instead of the current code
> 
> ```powershell
> echo "Installing $msi_path ..."
> Start-Process msiexec.exe -Wait -ArgumentList "/qn /norestart /i `"$msi_path`""
> echo "- Done"
> ```
> 
> the script should have something like: https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/Splunk.SignalFx.DotNet.psm1#L52-L57

@pjanotti is correct.  Currently, the script does not fail even if the msi installation is unsuccessful (instead, the script fails at the config file check since it didn't get installed).  The script needs to be updated to capture and check the exit code from `Start-Process` as @pjanotti suggested, such that the config file check only fails if the msi was installed successfully but did not contain a valid config file (which I can do in a separate PR).

Reply:
> Can we leave the suggested change as is then?

@theletterf I would rather leave the text without mentioning the admin rights and let @jeffreyc-splunk follow-up with a more general fix.
```suggestion
        throw "Valid Collector configuration file not found."
```